docs/guide/installation.md

```diff
@@ -0,0 +1,10 @@
+First, install [Roact](https://github.com/Roblox/Roact) and [Rodux](https://github.com/Roblox/Rodux) into the same place you intend to install Roact-Rodux. For these examples, we'll install everything into `ReplicatedStorage`.
+
+### Method 1: Model File (Roblox Studio)
+* Download the `rbxmx` model file attached to the latest release from the [GitHub releases page](https://github.com/Roblox/Roact-Rodux/releases)
+* Insert the model into Studio into a place like `ReplicatedStorage`
+
+### Method 2: Rojo
+* Copy the `lib` directory into your codebase
+* Rename the folder to `RoactRodux`
+* Use [Rojo](https://github.com/LPGhatguy/rojo) to sync the files into a place
```

Comment:
Are these instructions confusing given that the whole repo has Rojo support?  Or is this to divorce it from developing, where you might also want examples or the like to be hosted via the main repo's rojo.json?

Reply:
I'm not very satisfied with our installation instructions. In the not-too-distant-future, I'd like to be able to say "just use `rojo install github:Roblox/rodux` or at least point people in the direction of Git submodules.